performance_tests/test_perf_circleci.py

```diff
@@ -0,0 +1,98 @@
+"""
+This script takes in two CircleCI build numbers
+(usually the `build_python2` and `build_python2` of the same workflow)
```

Comment:
'build_python2' and 'build_python3'?

Reply:
Déplacer ce fichier dans `openfisca_france/scripts/` ?
Et supprimer `test_perf_tests.py` qui a été traité par #1010 ?